res/fm/rms/rms_run.py

```diff
@@ -112,6 +112,7 @@ def run(self):
         if os.path.isfile(exec_env_file):
             with open(exec_env_file) as f:
                 user_env = json.load(f)
+        print("USERENV", user_env)
```

Comment:
Do we really need these print statements?

Reply:
nevermind I see they are removed later on please drop this commit before rebasing

Reply:
nope ;)

Reply:
Other than that it looks good to me